lib/utils/scheduler.js

```diff
@@ -1,16 +1,18 @@
 const moment = require('moment');
 
-module.exports = (interval = 'day') => {
-
+module.exports = (interval = 'minute') => {
```

Comment:
can we make this an hour? It just bugs me that we use this for exactly one thing, and that runs hourly, so default should probably be set to the same value

Reply:
My reasoning here was to default to the same interval we use for the `setInterval` timing - if we do use it to schedule anything else in future then defaulting to `hour` or `day` might not make sense, so it's best to pass it in.

Reply:
at the moment we have a `default` value which is based on an imaginary future use care - in current software we _never_ want it to be a minute, setting a default to something we never want seems odd, maybe just remove the default and always pass it in